### DISCUSSION TITLE
Auto Deploy Rowy Run Error

### DISCUSSION THREAD

**Discussion by mlep1** [AntlerVC/firetable#871] [Support / Q&A] [answered]
Error: Google Cloud Run Service Agent does not have permission to get access tokens for the service account rowy-backend@projectID.iam.gserviceaccount.com. Please give service-13numbers@serverless-robot-prod.iam.gserviceaccount.com permission iam.serviceAccounts.getAccessToken on the service account. Alternatively, if the service account is unspecified or in the same project you are deploying in, ensure that the Service Agent is assigned the Google Cloud Run Service Agent role roles/run.serviceAgent.

Havent skipped any steps in the deployment documentation - how to resolve this error?

**Comment by BeeBombshell** [comment, marked as answer]
Hey @mlep1. Seems like you need to enable the permissions to the GCP service account. This could happen if your Cloud Run Service Agent is missing.
Try creating a service account as follows:
```
Name: service-<account-id>@serverless-robot-prod.iam.gserviceaccount.com
Role: Google Cloud Run Service Agent
```

Or if this doesn't work, try enabling permissions to the Cloud Build service account. By navigating to:
Google Cloud Console > Cloud Build > Cloud Build Settings > enable access to the GCP services that you are using.

**Reply by mlep1** [reply]
Thanks @BeeBombshell 

When I attempt to add the service account I get the error "Email addresses and domains must be associated with an active Google Account, Google Workspace account or Cloud Identity account"

**Reply by BeeBombshell** [reply]
Hey @mlep1, has this issue been fixed?

**Reply by BeeBombshell** [reply]
Hey @mlep1, closing this discussion since it has been unattended for a long time. If you're still facing this issue, feel free to create a new one, or [book a short 1:1](https://rowy.io/meet) so that we can go over this and get it resolved.
